plugins/check_snmp_storage.pl

```diff
@@ -217,29 +242,29 @@ sub help {
    <type>: Make the output shorter :
      0 : only print the global result except the disk in warning or critical
          ex: "< 80% : OK"
-     1 : Don't print all info for every disk
+     1 : Don't print all info for every disk 
          ex : "/ : 66 %used  (<  80) : OK"
    <where>: (optional) if = 1, put the OK/WARN/CRIT at the beginning
    <cut>: take the <n> first caracters or <n> last if n<0
 -o, --octetlength=INTEGER
   max-size of the SNMP message, usefull in case of Too Long responses.
   Be carefull with network filters. Range 484 - 65535, default are
-  usually 1472,1452,1460 or 1440.
+  usually 1472,1452,1460 or 1440.   
 -t, --timeout=INTEGER
```

Comment:
Can you please avoid such changes not related to functional changes?

Reply:
I'm sorry - I started with a perltidy using the supplied .perltidyrc